pkg/tsdb/tempo/trace.go

```diff
@@ -28,7 +28,11 @@ func (s *Service) getTrace(ctx context.Context, pCtx backend.PluginContext, quer
 		return nil, err
 	}
 
-	request, err := s.createRequest(ctx, dsInfo, model.Query, query.TimeRange.From.Unix(), query.TimeRange.To.Unix())
+	if model.Query == nil || *model.Query == "" {
+		return result, fmt.Errorf("trace id is required")
```

Comment:
This error message should be updated to reflect that this can be a trace id or a traceql query.